src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo | null;
+  setVisible: React.Dispatch<React.SetStateAction<boolean>>;
+  setTodo: React.Dispatch<React.SetStateAction<Todo | null>>;
+};
+
+export const TodoModal: React.FC<Props> = ({ setVisible, todo, setTodo }) => {
+  const [isLoad, setIsLoad] = useState(true);
```

Comment:
The state variable `isLoad` is not named according to common conventions. Consider renaming it to `isLoading` for better readability and to follow the convention of using present participles for boolean state variables.